### ISSUE TITLE
[Request] GDScript rewrite

### ISSUE THREAD

**Issue by PunnyBunny1** [theRealUnd3rdog/Godot_Noshot_Controller#1]
First, I want to say thank you for offering this for free.

Is it possible to make this using GDScript instead? 

I've also noticed that it doesn't run very smoothly on my computer. Sometimes it's as slow as 20 frames per second, and other times it's faster at 70 frames per second, but there are occasional jumps/spikes in performance. Is there anything I can do to make it run better, or do I need to get a new computer?

**Comment by theRealUnd3rdog** [comment]
Yeah, I've been thinking of getting a friend of mine to convert it to
GDScript, but currently out of time I won't be able to.

If the lag spikes are prominent it could be cause of the SDFGI and some of
the graphics going on in there. I didn't totally optimize stuff from that
end, so I recommend you turn off some of the settings as I cranked a lot of
them up.

It's also using TAA, so that might be another reason why you get the frame
drops. The post processing should be cheap though.

**Comment by zikaura** [comment]
@theRealUnd3rdog I'm actually working on a GDScript rewrite for this. Can I make a PR when I'm done?

**Comment by theRealUnd3rdog** [comment]
> @theRealUnd3rdog I'm actually working on a GDScript rewrite for this. Can I make a PR when I'm done?

Sure you can! Just make sure to have a separate branch for GDScript as I'll still be further developing this in C#.

**Comment by notbojo** [comment]
> [@theRealUnd3rdog](https://github.com/theRealUnd3rdog) I'm actually working on a GDScript rewrite for this. Can I make a PR when I'm done?

@zikaura  hey, have you finished working on the rewrite? rly need it rn.
